test/Cake.MinVer.Tests/MinVerSettingsTests.cs

```diff
@@ -70,14 +71,15 @@ public void Should_Shallow_Clone_All_Properties()
             actual.NoWorkingDirectory.Should().Be(expected.NoWorkingDirectory);
             actual.ArgumentCustomization.Should().Be(expected.ArgumentCustomization);
             actual.EnvironmentVariables.Should().BeEquivalentTo(expected.EnvironmentVariables);
+            actual.HandleExitCode.Should().BeEquivalentTo(expected.HandleExitCode);
 
             var properties = typeof(MinVerSettings)
                 .GetProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.FlattenHierarchy)
                 .Select(p => p.Name)
                 .ToList();
 
             // Sanity check & alarm to detect changes in properties that need to be considered in the cloning
-            properties.Should().HaveCount(17);
+            properties.Should().HaveCount(18);
```

Comment:
We should replace this sanity check with a proper API snapshot approval test.

Some possibilities:
- [Verify](https://github.com/VerifyTests/Verify)
- [ApprovalTests](https://github.com/approvals/ApprovalTests.Net)